lib/ntpUtil.js

```diff
@@ -4,7 +4,35 @@
 
 const childProcess = require('child_process')
 const fs = require('fs')
+const path = require('path')
 const util = require('../lib/util')
+const request = require('request')
+
+const createPhotoJsonFile = (path, body) => {
+  fs.writeFileSync(path, body)
+}
+
+const getImageFileNameListFrom = (photoJsonObj) => {
+  let fileList = []
+  if (photoJsonObj.logo)
+    fileList.push(photoJsonObj.logo.imageUrl)
+
+  if (photoJsonObj.wallpapers) {
+    photoJsonObj.wallpapers.forEach((wallpaper) => {
+      fileList.push(wallpaper.imageUrl)
+      if (wallpaper.logo && wallpaper.logo.imageUrl) {
```

Comment:
maybe a comment above this - saying `// V2 feature - support per-wallpaper logo`

Reply:
Added.

Reply:
Added.